### DISCUSSION TITLE
How can I relate ncase and the output dose of different medical equimpment?

### DISCUSSION THREAD

**Discussion by kstcmcjw** [nrc-cnrc/EGSnrc#881] [Questions] [answered]
Hello everyone, wish you have a nice day!
First, i am apologise for my so many questions.
As the MC don't simulate the time, we can't simulate the dose-rate, but i really want to establish a relationship between clinc radiology and our EGS modeling.
The ncase=10^n, it is stand for the photon or electron hit the target area's amount.
I know the acceleator, and a EGS guider @rtownson answered me to  consider relative comparisons with MC than seek absolute dose, and need to calibrate the MC simulation with experiment, er, i think it can establish a relationship between MU and the ncase, thanks to @rtownson very much.
But in the DR and CT, how to simulate the exposure aparament of mA, s, or mAs ?   How to simulate the DR and CT output dose?

Thank you for your attention!

**Comment by ftessier** [comment, marked as answer]
In practice it is difficult to establish a direct correspondence between MC and experiment, because in Monte Carlo ultimately the only "time" metric you have is the number of primary particles (number of histories), but experimentally you don't know this value precisely.

If you model the incident electrons that strike the target to generate the x-rays then in principle you can relate _that_ directly to the tube current, however in practice I don't recall ever matching this to better than about 20% in x-ray tubes and linacs.

Therefore, in order to calibrate the "Monte Carlo time", you ought to perform a calibration simulation, which typically involves modelling a detector and comparing with an experimental measurement (for example model the monitor chamber which provides monitor units). But even that is not trivial. The next best thing is to rely on result **ratios** in simulations, to normalize out quantities that are not known or not know precisely enough.

**Reply by kstcmcjw** [reply]
Thanks for your such a detail answer! It seems that EGSnrc and BEAM are very suitable for the dose deposit estimate, especially for the absorb-dose or the organ-dose,  well, thanks to @ftessier, i will search some EGS and BEAM papers though the PUBmed, to reasearch which field they can make a good effect.

Thank you @ftessier, have a nice day !
